Source/IGListAdapter.m

```diff
@@ -272,6 +272,7 @@ - (void)performUpdatesAnimated:(BOOL)animated completion:(IGListUpdaterCompletio
     id<IGListAdapterDataSource> dataSource = self.dataSource;
     UICollectionView *collectionView = self.collectionView;
     if (dataSource == nil || collectionView == nil) {
+        IGLKLog(@"Warning: Your call to performUpdatesAnimated-method is ignored as dataSource or collectionView haven't been set.");        
```

Comment:
instead of "performUpdatesAnimated-method" we can use `%s` and pass `__PRETTY_FUNCTION__` 😄